app/src/js/utils/table-config/metrics.js

```diff
@@ -1,15 +1,129 @@
-// import React from 'react';
-// import { Link } from 'react-router-dom';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { shortDateNoTimeYearFirst } from '../../utils/format';
+import { daacOptionNames } from '../../selectors';
 
 export const tableColumns = [
   {
-    Header: 'Name',
-    accessor: (row) => row.event_type,
-    id: 'event_type'
+    Header: 'Event',
+    accessor: (row) => row.id,
+    Cell: row => <Link to={{ pathname: `/metrics/id/${row.row.original.id}` }} aria-label="View your event details">{row.row.original.id}</Link>,
+    id: 'id'
   },
   {
     Header: 'Count',
     accessor: (row) => row.count,
     id: 'count'
   }
 ];
+
+export const getTime = (obj) => {
+  if (typeof obj === 'undefined') return '';
+  let time = '';
+  if (obj.hours) {
```

Comment:
Unfortunately, this time object returned can be expressed as `years`, `mons`, and `days` as well. @DavisGauntUAH we should probably switch the metrics endpoint from `AGE()` to `EXTRACT(EPOCH FROM ())` to instead just return the number of seconds between the two timestamps.
Additional details:
https://www.postgresql.org/docs/current/functions-datetime.html#FUNCTIONS-DATETIME-EXTRACT
https://stackoverflow.com/a/14020951

Reply:
I can update the v2 endpoint to do that early next week before its merged